### PR TITLE
Feat/#284 style scrollbar

### LINKED ISSUE #284: Style the scrollbar for message list.
The default scrollbar in the message list panel currently maintains the browser's default style. It is functional but doesn't align with the application's overall aesthetic. Implementing a custom style for the scrollbar will enhance the visual appeal and user experience, making the UI more consistent and engaging.

###### Expected Scrolbar Design
Expected Scrollbar design

### PR DESCRIPTION
**Pull request by umangutkarsh**
# Brief Title
The default scrollbar is styled to a custom one, with the help of _webkit-scrollbar_.

## Acceptance Criteria fulfillment

- [X] Added the css for custom scrollbar.
- [X] Include thr scrollbar styles directly in the `index.js` javascript file.

Fixes #284 

## Video/Screenshots
scroll
scroll2

### CODE REVIEW

**Comment by umangutkarsh** [comment]
Hey, I've made a PR. Please review it. Thanks for your time.

File: packages/react/src/index.js [hunk 1]
```diff
@@ -14,6 +14,35 @@ import { Box } from './components/Box';
...
+  /* Style the scrollbar track (the background) */
+  ::-webkit-scrollbar-track {
+    background: #ffffff;
+  }
+`;
+
+const style = document.createElement('style');
+style.type = 'text/css';
+style.innerHTML = scrollbarStyles;
+document.getElementsByTagName('head')[0].appendChild(style);
```
Comment (abhinavkrin) [inline packages/react/src/index.js:44]: When creating an NPM package that includes React components, it's crucial to provide styles in a way that they are encapsulated within the components themselves and do not unintentionally affect the global scope of the consuming application.

**Review by abhinavkrin — COMMENTED** [review]
When creating an NPM package that includes React components, it's crucial to provide styles in a way that they are encapsulated within the components themselves and do not unintentionally affect the global scope of the consuming application

**Comment by abhinavkrin** [comment]
Closing this PR as the issue has been fixed by #291

**Pull request closed without merging**